goldenmaster/testbed2_olink/sinks.py

```diff
@@ -162,13 +271,26 @@ def olink_object_name(self):
         return 'testbed2.NestedStruct3Interface'
 
     def olink_on_init(self, name: str, props: object, node: "ClientNode"):
+        self.client = ClientNode.register_sink(self)
         for k in props:
-            setattr(self, k, props[k])
+            if k == "prop1":
```

Comment:
should those changes on init also trigger notifications? other implementations do